docs/adr/0004-running-out-of-rancher-manager-cluster.md

```diff
@@ -0,0 +1,29 @@
+# 4. Running out of Rancher Manager cluster
+
+* Status: proposed
+* Date: 2023-08-31
+* Authors: @salasberryfin
+* Deciders: @richardcase @alexander-demicev @furkatgofurov7 @Danil-Grigorev @mjura
+
+## Context
+
+As an operator, I want the choice to deploy Rancher Turtles in the same or a different cluster to Rancher Manager, so that I have choice in my deployment topology.
```

Comment:
```suggestion
As an operator, I want the choice to deploy Rancher Turtles in the same or a different cluster to Rancher Manager, so that I have a choice in my deployment topology.
```